email-ballerina/pop_listener_endpoint.bal

```diff
@@ -101,10 +101,11 @@ public class PopListener {
     isolated function internalStart() returns @tainted error? {
         self.jobId = check task:scheduleJobRecurByFrequency(new PopJob(self), self.config.pollingInterval);
         log:printInfo("User " + self.config.username + " is listening to remote server at " + self.config.host + "...");
```

Comment:
IMO we should move these stdlibs info logs to debug level. Since we providing the standard library functionalities to ballerina developers, we should avoid such logs. it is better to let ballerina developers to add there own logs in their codes. ex: java standard libraries hardly print info level logs.

Reply:
Changed to debug logs